### ISSUE TITLE
Common check interval annotation

### ISSUE THREAD

**Issue by hazim1093** [stakater/IngressMonitorController#69]
As suggested by The annotation `monitor.stakater.com/pingdom/resolution` can be abstracted into a general annotation such as `monitor.stakater.com/resolutionTime` as it can be used with premium services of uptimerobot and probably other services providers too. 

Make it the annotation common and document that it is currently supported with pingdom only, we can later add support for resolution time for uptimerobot as well (Currently uptimerobot does not allow specifying resolution times manually for free services)

**Comment by drubin** [comment]
But it does support it easily with the premium version. 

https://uptimerobot.com/api#newMonitorWrap 

While we are abstracting away these thigns it might also be nice to abstract away the AlertContacts as well.  Different services might need to go to different contacts 

It also explains the confusing `_0_0` in the alert contacts configuration.

> alert_contacts - optional (the alert contacts to be notified when the monitor goes up/down.Multiple alert_contact>ids can be sent like alert_contacts=457_0_0-373_5_0-8956_2_3 where alert_contact>ids are seperated with - and threshold + recurrence are seperated with _. For ex: alert_contacts=457_5_0 refers to 457 being the alert_contact>id, 5 being the threshold and 0 being the recurrence. As the threshold and recurrence is only available in the Pro Plan, they are always 0 in the Free Plan)

**Comment by rasheedamir** [comment]
Agreed @drubin; can you please create a separate issue and suggest/mention the abstracting of AlertContacts in it?

**Comment by karl-johan-grahn** [comment]
Reopening issues that inadvertently were closed as stale
